lib/beetle/configuration.rb

```diff
@@ -35,6 +35,12 @@ class Configuration
     # handler timeout.
     attr_accessor :redis_failover_timeout
 
+    # how long we want status keys to survive after we have seen the second message of
+    # a redundant message pair. Setting this to a high value (hours) will reduce the
+    # likelihood of executing handler logic, but can cause a higher redis database
```

Comment:
Still missing the "executing handler logic **more than once**" part. 

I would also add a short description why seeing a message more than twice could happen (e.g. due to a RabbitMQ redelivery or the publisher sending more than two copies).